LBH.AdultSocialCare.Api/V1/Gateways/Interfaces/IResidentialCarePackageGateway.cs

```diff
@@ -9,5 +9,7 @@ namespace LBH.AdultSocialCare.Api.V1.Gateways.Interfaces
     public interface IResidentialCarePackageGateway
     {
         public Task<ResidentialCarePackage> UpsertAsync(ResidentialCarePackage residentialCarePackage);
+
+        public Task<ResidentialCarePackage> GetAsync(Guid residentialCarePackageId);
```

Comment:
More descriptive names for functions here too. `UpsertResidentialCarePackageAsync()`, `GetResidentialCarePackageAsync()`